docs/create_rules.md

```diff
@@ -90,3 +90,21 @@ def sql_has_reasonable_number_of_lines(model: Model, max_lines: int = 200) -> Ru
             message=f"SQL query too long: {count_lines} lines (> {max_lines})."
         )
 ```
+
+### Filtering, skipping and expanding
+
+It's possible to modify existing rules, either built-in or custom.
+A possible use case is skipping a default rule for certain models.
+Remember to also disable the original rule (see [Configuration](configuration.md)).
```

Comment:
I'm quite on the fence with this approach. It requires a 2-step process to be followed, disabling a rule and creating a wrapper for it, which is quite involved and complex.

I understand you explored the alternative approach of evaluating an expression stored in the config file. Was there any blocker with it?

Reply:
The "blocker" is my unwillingness to use `eval()` in the code, really. Safely parsing an expression like `skip_if = "(model.schema.lower() == 'x') or (model.schema.lower() == 'y' and model.access == 'public')` is quite a large task.

If we just `eval()` it is of course not hard to implement. Which to be fair, is not really that much unsafer than using user-defined Python code. But at least from an user perspective, it is clearly Python code and not some configuration file.

Reply:
Hey @diorge, first of all thanks for opening the PR :raised_hands: I'd like to pitch in a little bit on the design of things here. I agree with both of you: using `eval` is not too nice and I would love users to write python code in a python file (instead of a config file). :grin: However, this solution is also not perfect IMO, because it requires users to recreate a rule in order to configure skipping. It would be great if we could just add skip configuration to existing rules, without having to recreate them and at the same time write the skip logic in Python.

I think I have an idea that would solve all concerns, please think along! What if we do the following: we implement something similar to `@rule`; let's call it `@skip` for now. `@skip` will be discovered in a similar way as `@rule` and can be defined by users themselves like this:

```python
from dbt_score import Model, skip

@skip
def temp_schema(model: Model) -> bool:
    """Skip models in the temp schema"""
    if model.schema == "temp":
        return True
    return False
```

By doing this, the skip logic is entirely written in Python, and it follows the approach we take for rules! So it's quite easy IMO.

Then, to configure the skips, they can be configured on rule level and re-used! We could configure like this:

```toml
[tool.dbt-score.rules."generic.has_description"]
skips = ["temp_schema"]
```

I think this would solve the biggest concerns we have with the current two proposal:
- Not writing Python in python files
- No use of `eval()`
- Not having to disable + recreate rules to add skip logic
- Not being able to re-use skip-logic in all rules

This will be harder to implement, but it's the UX that counts if you ask me! Please let me hear your opinions! Happy to collaborate on this by the way